Feat/cqrs detection step

amoya87 opened the pull request:
Consume, corrections, and productions

amoya87 committed:
PrvCandidates

amoya87 committed:
Correction

amoya87 committed:
Remove PrvCandidates from detection_step

amoya87 committed:
Produce kafka messages

amoya87 committed:
detection_step   Settings File

pgallardor reviewed (commented):
Creo que la estructura del codigo esta super!
Esperemos la review de Diego y hacemos el merge

dirodriguezm committed:
chore: delete unused script

dirodriguezm committed:
build: fix image cmd

dirodriguezm committed:
refactor: prv_candidates are no longer present

previous candidates are deleted on the prv_candidates step, so the
messages consumed by detection_step won't have that field.
